modules/dists/BlockDist.chpl

```diff
@@ -1288,10 +1288,6 @@ proc BlockArr.doiCanBulkTransfer(viewDom) {
     for param i in 1..rank do
       if viewDom.dim(i).stride != 1 then return false;
 
-  // See above note regarding aliased arrays
```

Comment:
Is there a note above to remove as well?

Reply:
There is a config const that could be removed called `disableAliasedBulkTransfer`. It's only used in a couple of compopts and in miniMD's release Makefile. Removing it with this PR would not change the behavior of those programs.

Reply:
Seems like we should remove it soon though, right?

Reply:
I believe we should, yes. Any objections?

Reply:
I'm happy with removing it, if it is no longer used after this PR. But you could leave that to another PR if you like.

Reply:
I'll leave it for another PR for now, and give @bradcray a chance to chime in.